tensorflow/lite/testing/op_tests/pad.py

```diff
@@ -63,23 +66,52 @@ def make_pad_tests(options):
                        [[0, 1], [0, 0], [0, 0], [2, 3]],
                        [[0, 0], [0, 0], [0, 0], [0, 0]]],
           "constant_paddings": [True],
-          "fully_quantize": [True]
+          "fully_quantize": [True],
+          "quant_16x8": [False]
```

Comment:
Is there a reason we can't use [False, True] here for quant_16x8? Rather than duplicating the test? Which parameters are different?